emmet-core/emmet/core/molecules/bonds.py

```diff
@@ -0,0 +1,323 @@
+import logging
+from collections import defaultdict
+from typing import Dict, List, Any, Tuple, Union
+import copy
+
+from typing_extensions import Literal
+
+import numpy as np
+from pydantic import Field
+import networkx as nx
+
+from pymatgen.core.structure import Molecule
+from pymatgen.analysis.graphs import MoleculeGraph
+from pymatgen.analysis.local_env import OpenBabelNN, metal_edge_extender
+
+from pymatgen.core.periodic_table import Specie, Element
+
+from emmet.core.mpid import MPID
+from emmet.core.qchem.task import TaskDocument
+from emmet.core.material import PropertyOrigin
+from emmet.core.molecules.molecule_property import PropertyDoc
+
+
+metals = ["Li", "Mg", "Ca", "Zn", "Al"]
+
+
+def fix_C_Li_bonds(critic):
+    for key in critic["bonding"]:
+        if critic["bonding"][key]["atoms"] == ["Li","C"] or critic["bonding"][key]["atoms"] == ["C","Li"]:
+            if critic["bonding"][key]["field"] <= 0.02 and critic["bonding"][key]["field"] > 0.012 and critic["bonding"][key]["distance"] < 2.5:
+                critic["processed"]["bonds"].append([int(entry) - 1 for entry in critic["bonding"][key]["atom_ids"]])
+    return critic
+
+
+def make_mol_graph(mol, critic_bonds=None):
+    mol_graph = MoleculeGraph.with_local_env_strategy(mol,
+                                                      OpenBabelNN())
+    mol_graph = metal_edge_extender(mol_graph)
+    if critic_bonds:
+        mg_edges = mol_graph.graph.edges()
+        for bond in critic_bonds:
+            bond.sort()
+            if bond[0] != bond[1]:
+                bond = (bond[0], bond[1])
+                if bond not in mg_edges:
+                    mol_graph.add_edge(bond[0],bond[1])
+    return mol_graph
+
+
+def nbo_molecule_graph(
```

Comment:
Is there a reason you chose to make these module level functions instead of methods on `BondingDoc`? Do you intend on reusing them elsewhere?